foundry.toml

```diff
@@ -16,7 +16,7 @@ optimizer_runs = 200
 # Whether or not to use the Yul intermediate representation compilation pipeline
 via_ir = false
 # Override the Solidity version (this overrides `auto_detect_solc`)
-solc_version = '0.8.12'
+solc_version = '0.8.27'
```

Comment:
Mentioned in other PR, but should we update minimum pragma solidity versions in each file too?
Any solc version less than 0.8.27 will not compile with our custom error changes